src/DAL/migration/FullSchema.sql

```diff
@@ -85,13 +87,13 @@ CREATE INDEX IF NOT EXISTS record_id_idx
 
 CREATE INDEX IF NOT EXISTS source_date_end_idx
     ON "PolygonParts".parts USING btree
-    ("sourceEndDateUtc" ASC NULLS LAST)
+    ("sourceDateEnd" ASC NULLS LAST)
     TABLESPACE pg_default;
 -- Index: source_date_start_idx
 
 -- DROP INDEX IF EXISTS "PolygonParts".source_date_start_idx;
 
 CREATE INDEX IF NOT EXISTS source_date_start_idx
     ON "PolygonParts".parts USING btree
-    ("sourceStartDateUtc" ASC NULLS LAST)
+    ("sourceDateStart" ASC NULLS LAST)
     TABLESPACE pg_default;
```

Comment:
Please add a line at the end of the file.

Reply:
done